debug_toolbar/static/debug_toolbar/js/history.js

```diff
@@ -7,34 +7,46 @@ $$.on(djDebug, "click", ".switchHistory", function (event) {
     const newStoreId = this.dataset.storeId;
     const tbody = this.closest("tbody");
 
-    tbody
-        .querySelector(".djdt-highlighted")
-        .classList.remove("djdt-highlighted");
-    this.closest("tr").classList.add("djdt-highlighted");
+    const highlighted = tbody.querySelector(".djdt-highlighted");
+    if (highlighted) {
+        highlighted.classList.remove("djdt-highlighted");
+        this.closest("tr").classList.add("djdt-highlighted");
```

Comment:
I think this line needs to be pulled out of the if statement. Otherwise if nothing is highlighted from the beginning, the if statement can never be true.